mne/preprocessing/maxwell.py

```diff
@@ -565,8 +570,10 @@ def _overlap_projector(data_int, data_res, delta=0.02):
     C_mat = np.dot(Q_int.T, Q_res)
 
     # Compute angles between subspace and which bases to keep
-    _, S_intersect, Vh_intersect = svd(C_mat)
+    _, S_intersect, Vh_intersect = svd(C_mat, full_matrices=False)
     retain_inds = S_intersect >= (1 - delta)
+    logger.info('    Projecting out %s tSSS components for %s-%s'
+                % ((~retain_inds).sum(), tlims[0], tlims[1]))
```

Comment:
This says that 53 and 46 components are rejected, by the way. This seems like way too many...